KLSadd_insertion/src/linetest.py

```diff
@@ -1,10 +1,7 @@
-__author__ = "Rahul Shah"
-__status__ = "Development"
-
 """
```

Comment:
All comments should be relevant to the code and actually helpful - if it is easy to infer what the like of code does without the comment, get rid of the comment

Also, comments should always have a space after the `#` symbol.